aws-iot-mitigationaction/inputs/inputs_1_create.json

```diff
@@ -0,0 +1,15 @@
+{
+  "ActionName": "CfnContractTest",
+  "RoleArn": "{{RoleForDeviceDefenderAuditArn}}",
```

Comment:
Let's not use "Audit" in names of even the test resource. Reason being MitigationActions isn't specific to Audit anymore (it's a common resource to both Audit/Detect)

Reply:
Removed